library/network/src/modules/SuSEFirewall.rb

```diff
@@ -1151,6 +1151,92 @@ def GetInterfacesInZoneSupportingAnyFeature(zone)
       GetInterfacesInZone(zone)
     end
 
+    # Function sets status for several services in several firewall zones.
+    #
+    # @param	list <string> service ids
+    # @param	list <string> firewall zones (EXT|INT|DMZ...)
+    # @param	boolean new status of services
+    # @return	[Boolean] if successfull
+    #
+    # @example
+    #	SetServicesForZones (["samba-server", "service:irc-server"], ["DMZ", "EXT"], false);
+    #	SetServicesForZones (["samba-server", "service:irc-server"], ["EXT", "DMZ"], true);
+    #
+    # @see #GetServicesInZones()
+    # @see #GetServices()
+    def SetServicesForZones(services_ids, firewall_zones, new_status)
+
+      Yast.import "SuSEFirewallServices"
+
+      services_ids = deep_copy(services_ids)
+      zones = deep_copy(firewall_zones)
+      # no groups == all groups
+      if zones.empty?
+        zones = GetKnownFirewallZones()
+      end
+
+      # Rebuild services since we may have more than one FirewallD
+      # service per SF2 one
+      if self.class == SuSEFirewalld
```

Comment:
I think this is not necessary because this method is part of the SuSEFirewalld class.